build.gradle

```diff
@@ -1195,8 +1195,10 @@ project(':connect:runtime') {
   archivesBaseName = "connect-runtime"
 
   dependencies {
+    compileOnly project(':connect:json')
+    compileOnly project(':connect:transforms')
+
```

Comment:
Seems strange to change the order, even if Gradle doesn't care. API is far more important, and IMO should be first, even if these are `compileOnly`.